jsx/App/Stories/Story/Display/TextFormatResultWindow.jsx

```diff
@@ -0,0 +1,290 @@
+                }
+                // Reason for using the replace with "_" is that some glossed word is two words in
+                // the translation, but two words with a space in between will be recognized as two
+                // separate glossed word by the LaTeX package, so adding the underscore makes sure 
+                // that a phrase made up with multiple words can still be grouped together after being rendered in LaTeX. 
+                glossList.push(glossForThisWord.join("").replace(" ", "\\_"));
+            }
+        }
+        glossList.push("\\\\ \n  ");
+
+        return glossList.join(" ");
+    }
+
+    /* Puts the sentence translation into LaTeX format. */
+    getSentenceTranslationLine(sentenceObject) {
+        const sentence = sentenceObject[0]["value"];
+        const translationStart = "\\glt `";
+        const translationEnd = "' \\\\ \n  ";
+        return translationStart + sentence + translationEnd;
+    }
+
+    /* Checks if an item is a suffix or clitic. */
+    isSuffix(item) {
+        // Suffix or clitic starts with = or -, or the entire word is capitalized.
+        return item.startsWith("=") || item.startsWith("-") || item === item.toUpperCase();
+    }
+
+    /* Displays the created material in a popup window. */
+    displayResult(processedMaterial, latexLines) {
+        const headerLine = "Format result: ";
+        const storyTitleLine = "Story title: " + processedMaterial["title"].replace(/\_/g, " ") + "\n"; 
+        const storyIdLine = "Story ID: " + processedMaterial["storyId"].replace(/_/g, "\\_") + "\n"; 
+        const sentenceUrlLine = "Sentence URL: " + processedMaterial["sentenceUrl"].replace(/_/g, "\\_") + "\n"; 
+        const latexLibraryLine = "Formatted for gb4e and gb4e-modified LaTeX packages: ";
+
+        let resultContainer;
+        let resultContainers = document.getElementsByClassName("formatResultContainer");
+        for (var e of resultContainers) {
+          if (e.getAttribute("sentenceId") == this.props.sentenceId) {
+            resultContainer = e;
+            break;
+          }
+        }
+        this.createParagraphElement(resultContainer, headerLine); 
+        this.createParagraphElement(resultContainer, storyTitleLine); 
+        this.createParagraphElement(resultContainer, storyIdLine); 
+        this.createParagraphElement(resultContainer, sentenceUrlLine); 
+        this.createParagraphElement(resultContainer, latexLibraryLine); 
+        this.createParagraphElement(resultContainer, latexLines); 
+    }
+
+    /* Creates an HTML element <pre>, which means that its content is pre-formatted text. */
+    createParagraphElement(resultContainer, content) {
+      const newParagraphElement = document.createElement("pre");
+      newParagraphElement.innerHTML = content;
+      resultContainer.appendChild(newParagraphElement);
+    }
+
+    /* Reload the window when the close button is clicked. This clears the tier selection window and result window. */
```

Comment:
A note for posterity: the "close" button is needed because currently, after the user clicks Confirm and sees the result, there’s no way to change what this result is. So if the user decides to change what buttons are selected and clicks Confirm again, the result won’t change; they need to Close (i.e. refresh) first. 

We should hide the Confirm button after it's clicked, since it can't to anything anymore.